pkg/il/evaluator/evaluator.go

```diff
@@ -32,9 +34,17 @@ import (
 // IL is an implementation of expr.Evaluator that also exposes specific methods.
 // Specifically, it can listen to config change events.
 type IL struct {
+	cacheSize       int
+	attrContext     *attrContext
```

Comment:
up to you, but I might consider just calling the field `context`.  The `attr` is almost implied.

Reply:
Done